spec/lib/extensions/brakeman_fingerprint_patch_spec.rb

```diff
@@ -41,127 +56,219 @@ def location = {}
   context "running from the core repo" do
     context "with a git-based engine" do
       context "in the system gem location" do
-        let(:engine_path) { "/Users/user/.gem/ruby/3.1.5/bundler/gems/manageiq-ui-classic-df1d9535ef51/" }
```

Comment:
ah, those pesky trailing slashes must have messed up your sort_by on `/`

Reply:
actually, what changed was moving to Pathname (which is what `.root` will return) - once we have Pathname, those trailing slashes are unnecessary (and actually break things).